crates/astria-conductor/src/sequencer/mod.rs

```diff
@@ -285,35 +297,244 @@ async fn subscribe_new_blocks(
     .map_err(|_| ResubscriptionError::BackoffFailed)
 }
 
-fn forward_pending_block(
+/// Forwards a sequencer block to the executor if it contains the expected height, or reschedules
+/// the block for later while fetching blocks for all missing heights.
+///
+/// The following cases are considered:
+///
+/// 1. if `h == h'` the block is forwarded to the executor. `h+1` is returned as the next expected
```

Comment:
can we change these to not be `h` and `h` but `h` and something else :p same for rest of the comment

Reply:
It's `h'` read as h-prime. That's what I grew up with 😭

I can see how it's tough to read depending on the font. I'll change to `k`.

Reply:
ah sorry i didn't see the `'`!!